devtools/environment.yml

```diff
@@ -54,5 +56,5 @@ dependencies:
   - tox                 # dev
   - twine               # dev
   - xlsxwriter          # base
-  - pip:
```

Comment:
Was this an make-the-environment-simpler hack? The rmi repo uses this package, so I'd like to keep it in here.

Reply:
This was just to keep it from outputting a bunch of pip messages whenever I update the environment. Didn't mean to leave in in the commit. Though I don't know what the consequence of depending on packages outside of conda will next time we try and release our software.